etc/custom.properties

```diff
@@ -113,16 +113,16 @@ org.opencastproject.download.directory=${org.opencastproject.storage.dir}/downlo
 
 # Opencast comes with the jdbc drivers for MariaDB (org.mariadb.jdbc.Driver) and PostgreSQL (org.postgresql.Driver).
 # To add other jdbcDrivers to the Opencast runtime, rebuild the db module with your desired drivers.
-#org.opencastproject.db.jdbc.driver=org.mariadb.jdbc.Driver
+org.opencastproject.db.jdbc.driver=org.mariadb.jdbc.Driver
 
 # The jdbc connection url, username, and password
 # Defaults:
 #  .url=jdbc:h2:${org.opencastproject.storage.dir}/db
 #  .user=sa
 #  .pass=sa
-#org.opencastproject.db.jdbc.url=jdbc:mariadb://localhost/opencast?useMysqlMetadata=true
-#org.opencastproject.db.jdbc.user=opencast
-#org.opencastproject.db.jdbc.pass=dbpassword
+org.opencastproject.db.jdbc.url=jdbc:mariadb://localhost/opencast?useMysqlMetadata=true
+org.opencastproject.db.jdbc.user=opencast
+org.opencastproject.db.jdbc.pass=dbpassword
```

Comment:
This probably needs to be reverted?

Reply:
Oh. Yes, this should be reverted. It was left from debugging.